cmd/openshift-install/create.go

```diff
@@ -326,6 +326,7 @@ func waitForInitializedCluster(ctx context.Context, config *rest.Config) error {
 	clusterVersionContext, cancel := context.WithTimeout(ctx, timeout)
 	defer cancel()
 
+	failing := configv1.ClusterStatusConditionType("Failing")
```

Comment:
Is this just until the CVO moves to Degraded?

Reply:
no plans for CVO to move to `Degraded`

Reply:
We're just going to translate operator Degradeds into CVO Failings forever?

Reply:
> no plans for CVO to move to `Degraded`
> We're just going to translate operator Degradeds into CVO Failings forever?
`forever` i can't say ;)
for now yes
`We're` the installer only cares about the fact that CVO reports failing, its only a consumer. the installer shouldn't care

Reply:
every team was told to move. why didn't the cvo move?